WalletWasabi.Fluent/Behaviors/MainWindowPersistPositionBehavior.cs

```diff
@@ -0,0 +1,93 @@
+using Avalonia;
+using Avalonia.Controls;
+using ReactiveUI;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using WalletWasabi.Fluent.ViewModels;
+using System.Linq;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class MainWindowBindPositionBehavior : DisposingBehavior<Window>
+{
+	protected override void OnAttached(CompositeDisposable disposables)
+	{
+		if (AssociatedObject is null)
+		{
+			return;
+		}
+
+		Observable.FromEventPattern(AssociatedObject, nameof(Window.DataContextChanged))
+				  .Subscribe(_ => BindPosition(disposables))
+				  .DisposeWith(disposables);
+	}
+
+	private void BindPosition(CompositeDisposable disposables)
+	{
```

Comment:
Shouldn't we check if `AssociatedObject` is null here?

Reply:
@yahiheb done